packages/api/src/command/connected-user/get-connected-user.ts

```diff
@@ -77,3 +77,26 @@ export const getConnectedUserByTokenOrFail = async (
 
   return connectedUser;
 };
+
+export const getConnectedUsersByTokenOrFail = async (
+  provider: string,
+  token: string
+): Promise<ConnectedUser[]> => {
```

Comment:
@RamilGaripov and @Orta21: now that we're creating a similar function to `getConnectedUserByTokenOrFail` (returns the first one we find), I'm thinking if its not the time to review it.

My thinking is:
- `getConnectedUserByTokenOrFail` calls `findOne`, which means there might be other records and we return the first one - since that's the only key/filter and we use like `%xxx%`, _in theory_ we could return the wrong connected user;
- looking at usages of that function, we only use it in one place, for Withings;
- maybe we can keep the new one and have Withings deal with the exception situation where it returns more than one record?

Also, this solution for Fitbit, to deal with the same user connecting from diff cxs, shouldn't we replicate it to other providers, like Withings?

Reply:
We could raise a follow-up issue to address this comment, and fix for other providers (ie Withings) - so that we don't have to change code in other providers for this PR